fixtures/packages/iron-icon/expected/.travis.yml

```diff
@@ -22,8 +21,8 @@ addons:
     packages:
       - google-chrome-stable
 script:
-  - xvfb-run polymer test
+  - xvfb-run polymer-test --npm
   - >-
-    if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then polymer test -s 'default';
-    fi
+    if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then polymer-test --npm -s
+    'default'; fi
```

Comment:
is there a way to get js-yaml to not reformat the file so diffs are more targeted?

Reply:
Maybe, I'll take a look

Reply:
Ok, I can turn off lineWidth checking, and make sure that single item arrays are turned into a string.

Reply:
awesome!